src/common/types.cpp

```diff
@@ -1039,6 +1019,37 @@ struct EnumTypeInfoTemplated : public EnumTypeInfo {
 		source.ReadStringVector(values_insert_order);
 		return make_shared<EnumTypeInfoTemplated>(move(enum_name), move(values_insert_order));
 	}
+
+	bool Equals(ExtraTypeInfo *other_p) override {
+		if (!other_p) {
+			return false;
+		}
+		auto &other = (EnumTypeInfo &)*other_p;
+		if (catalog_entry && catalog_entry == other.catalog_entry) {
+			// If they are both pointing to the same catalog entry it's done.
+			return true;
+		}
+		// if size and name are equal we return true
+		if (other.values_insert_order.size() == values_insert_order.size() && other.enum_name == enum_name) {
```

Comment:
What is the rationale here? Why can't I create two enums in different schemas with the same names and the same number of elements (e.g., English and Spanish versions)?